.github/workflows/manifold.yml

```diff
@@ -166,10 +143,10 @@ jobs:
         submodules: recursive
     - uses: jwlawson/actions-setup-cmake@v2
     - uses: ilammy/msvc-dev-cmd@v1
-    - name: Build ${{matrix.backend}}
+    - name: Build ${{matrix.parallel_backend}}
       shell: powershell
       run: |
-        cmake . -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DMANIFOLD_DEBUG=ON -DMANIFOLD_PAR=${{matrix.parallel_backend}} -A x64 -B build
+        cmake . -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DMANIFOLD_DEBUG=ON -DMANIFOLD_HULL=${{matrix.parallel_backend == 'NONE' && 'ON' || 'OFF'}} -DMANIFOLD_PAR=${{matrix.parallel_backend}} -A x64 -B build
```

Comment:
@kintel You can see here that our Windows CI is now building without Hull for the TBB version - everything compiles and links fine because the hull test file is excluded. You just get a linker error if you try to actually use the `Hull` functions with `MANIFOLD_HULL` disabled. Does that look like a reasonable way to use the library and minimize dependencies?